tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -697,6 +697,10 @@ Then(
   }
 )
 
+When('the user downloads a file of a previous version using the webUI', function () {
```

Comment:
```suggestion
When('the user downloads a previous version of the file using the webUI', function () {
```

Reply:
fixed
@pascalwengerter thank you fo review))